_config.yml

```diff
@@ -9,60 +13,72 @@ collections:
     title: Uploads
     output: true
 permalink: pretty
-author: The Debt Collective
-email: organizers@debtcollective.org
-description: The Debt Collective
+author:
+  name: The Debt Collective
+email: admin@debtcollective.org
+twitter:
+  username: 0debtzone
```

Comment:
0debtzone? Is that right? Besides, don't you need to add thee `@`

Reply:
The SEO plugin handles this for us